code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -493,6 +493,25 @@
 		dat += "<b>Already cast!</b><br>"
 	return dat
 
+/datum/spellbook_entry/summon/ghosts
+	name = "Summon Ghosts"
+	desc = "Spook the crew out by making them see dead people. Be warned, ghosts are capricious and occasionally vindicative, and some will use their incredibly minor abilties to frustrate you."
+	log_name = "SGH"

```

Comment:
considering the ghosts are now going to circle you for max valids you should give a point benefit ala summon magic/guns for this